packages/framework/src/Model/Product/Parameter/ParameterRepository.php

```diff
@@ -125,6 +125,22 @@ public function getAll(): array
             ->execute();
     }
 
+    /**
+     * @return \Shopsys\FrameworkBundle\Model\Product\Parameter\Parameter[]
+     */
+    public function getAllWithTranslations(): array
+    {
+        $this->em->createQueryBuilder()
+            ->select('p', 'pt')
+            ->from(Parameter::class, 'p')
+            ->join('p.translations', 'pt')
+            ->orderBy('p.id', 'asc')
+            ->getQuery()
+            ->execute();
+
+        return $this->getParameterRepository()->findBy([], ['id' => 'asc']);
```

Comment:
wow, this is an interesting approach. Executing the query loads the translations so they are then loaded in the following query? Do I understand it correctly? Is not it too magical? :smile: personally, I would prefer something like this - https://github.com/shopsys/shopsys/blob/15.0/packages/framework/src/Model/Payment/PaymentRepository.php#L154

Reply:
🙈  this is probably a mistake... I have changed it in bfd03e5c14